src/components/graph/BlendGraph.tsx

```diff
@@ -62,14 +70,18 @@ function BlendGraphLegend(props: BlendGraphLegendProps) {
     <LegendWrapper>
       <LegendItem>
         <LegendItemBox color={GRAY_STROKE_COLOR} />
-        <Text size='M' weight='medium' color={TEXT_COLOR}>Pool Returns</Text>
+        <Text size='M' weight='medium' color={TEXT_COLOR}>Blend Pool</Text>
```

Comment:
I recommend we add more weight to the Blend Pool to make it stand out more, but that is just my thought, feel free to keep it as is.